vllm/worker/habana_model_runner.py

```diff
@@ -1866,11 +1865,19 @@ def execute_model(
             raise ValueError(
                 "num_steps > 1 is not supported in HabanaModelRunner")
 
+        lora_mask: torch.Tensor = None
+        lora_logits_mask: torch.Tensor = None
         if self.lora_config:
             assert model_input.lora_requests is not None
             assert model_input.lora_mapping is not None
+            assert model_input.attn_metadata is not None
+            assert model_input.input_tokens is not None
+            assert model_input.lora_ids is not None
             self.set_active_loras(model_input.lora_requests,
```

Comment:
Consider separating out mask creating code into its own code-block, will make it easier to compare with model_runner code in future.